<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetMailEdit</class>
 <widget class="QWidget" name="WidgetMailEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>375</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Event</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  color: white;
  /*background-color: rgb(50,82,95);*/
  background-color: rgb(80,112,125);
}
QGroupBox {
  border-radius: 0px;
  border: 3px solid rgb(131, 147, 167);
  background-color: transparent; /*rgb(80,112,125);*/
}
QLineEdit {
  background-color: rgb(101,117, 137);
  border: 1px solid rgb(131, 147, 167);
}
QTextEdit {
  background-color: rgb(101,117, 137);  
}
QPlainTextEdit {
  background-color: rgb(101,117, 137);  
}
QComboBox {
  background-color: rgb(101,117, 137);  
}
QCheckBox::indicator {
  border: 2px solid rgb(80,112,125);
}
QCheckBox::indicator:checked {
  background-color: white;
}
QListView {
  background-color: rgb(101,117, 137);  
}
QTableWidget {
  background-color: rgb(101,117, 137);  
}
QDateTimeEdit {
  background-color: rgb(101,117, 137);  
}
QPushButton {
  background: rgb(101,117, 137);
}
QPushButton::checked {
  border: 2px solid white;
  font-weight: bold;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
        <property name="spacing">
         <number>9</number>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0,0,0,1">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>From:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditFrom">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="maxLength">
               <number>255</number>
              </property>
              <property name="placeholderText">
               <string>Sender</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonSend">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="focusPolicy">
               <enum>Qt::TabFocus</enum>
              </property>
              <property name="text">
               <string>Send</string>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>To:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="listWidgetTo">
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QScrollBar:horizontal {
 height: 4px;
 border: 1px solid rgb(131, 147, 167);
}
</string>
                </property>
                <property name="verticalScrollBarPolicy">
                 <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="horizontalScrollBarPolicy">
                 <enum>Qt::ScrollBarAsNeeded</enum>
                </property>
                <property name="verticalScrollMode">
                 <enum>QAbstractItemView::ScrollPerPixel</enum>
                </property>
                <property name="horizontalScrollMode">
                 <enum>QAbstractItemView::ScrollPerPixel</enum>
                </property>
                <property name="flow">
                 <enum>QListView::LeftToRight</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonSearchUser">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="toolTip">
                 <string>Address Book</string>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton {
  background-color: transparent;
  border-radius: 2px;
  border-image: url(:/icon-profile.png);
}
QPushButton:hover {
  background-color: rgb(50,82,95);
  border-radius: 2px;
}</string>
                </property>
                <property name="iconSize">
                 <size>
                  <width>16</width>
                  <height>16</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="labelDate">
                <property name="text">
                 <string>Date:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelSendDate">
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>-</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Subject:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditSubject">
              <property name="text">
               <string/>
              </property>
              <property name="maxLength">
               <number>255</number>
              </property>
              <property name="placeholderText">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>2</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>2</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
  background-color: rgb(131,147, 167);
}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="textEditBody">
            <property name="placeholderText">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditSubject</tabstop>
  <tabstop>textEditBody</tabstop>
  <tabstop>pushButtonSearchUser</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonSend</sender>
   <signal>clicked()</signal>
   <receiver>WidgetMailEdit</receiver>
   <slot>onBtnSendClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>664</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>737</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSearchUser</sender>
   <signal>clicked()</signal>
   <receiver>WidgetMailEdit</receiver>
   <slot>onBtnSearchUserClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBtnSendClicked()</slot>
  <slot>onBtnSearchUserClicked()</slot>
 </slots>
</ui>
